examples/multisig/hydrate.eg.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * @title Multisig Hydration
+ * @description Creates a multisig and encodes it into a hex string. This hex
+ * string is then used to hydrate a new MultisigRune. This is useful if you
+ * want to use it at a later date or if you want to send it to the other signatories.
```

Comment:
```suggestion
 * @description Create a multisig and encode its state into a hex string. Then use this
 * hex string to hydrate a new MultisigRune. This is useful if you want to share the
 * multisig setup info with the other signatories or a database.
```

Reply:
Final nit 😆

Reply:
Wouldn't it be better to keep the third person singulars? It sounds more descriptive -- without it they sound like instructions.

Reply:
Imo this / the imperative style is preferable

Reply:
I suppose we should decide on a consistent approach language we use, as with tsdocs... I could see it both ways. Hmm... what are your thoughts on why we would prefer the first way vs. the imperative way?

Reply:
> **Creates** a multisig and **encodes** its state into a hex string. Then **uses** this hex string to hydrate a new MultisigRune. This is useful if you want to share the multisig setup info with the other signatories or a database.

If we use third person singulars, it sounds like a proper description because its describing what the example is doing in the present tense.

Where as the latter sounds more like a set of instructions. For example if, I just add "please" in front:

> **Please** create a multisig and encode its state into a hex string. Then use this hex string to hydrate a new MultisigRune. This is useful if you want to share the multisig setup info with the other signatories or a database.

but this wouldn't work if they were third person singulars

> Please creates a multisig and encodes its state into a hex string. Then uses this hex string to hydrate a new MultisigRune. This is useful if you want to share the multisig setup info with the other signatories or a database.

Reply:
I can see the argument for the instructions style if we want to view the examples as a "problem-solution" format .

Reply:
Actually we'll just use the imperative format since everywhere else its already like that